ansible/playbook_Test_Scripts/Test_Script

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+printf "Enter the URL:  "			#User Input section
+read gitURL
+printf "Retain the VM? [y/n]\n" 
+read vmChoice
+
+
+cd ~/documents/					
+mkdir Unix_PB_Test_Folder
+						#Setup
+cd Unix_PB_Test_Folder
+mkdir Log_Files
+git clone $gitURL
+cd ${gitURL##*/}
+cd ansible
+
+for OS in Ubuntu1804 Ubuntu1604 CentOS6 	#For all tested OSs / Playbooks
+do
+	ln -sf Vagrantfile.$OS Vagrantfile	#Alias the correct vagrant file
+	vagrant  up
+	vagrant ssh -c "cd /vagrant/playbooks/AdoptOpenJDK_Unix_Playbook && sudo ansible-playbook --skip-tags "adoptopenjdk,jenkins" main.yml" 2>&1 | tee ~/documents/Unix_PB_Test_Folder/Log_Files/${gitURL##*/}.vagrant.$OS.log
```

Comment:
Naming conventions apply and also either comment this line or split it out so the intent is easier to grok.